spec/models/saved_claim/caregivers_assistance_claim_spec.rb

```diff
@@ -33,56 +31,67 @@
     end
 
     it 'calls PdfFill::Filler#fill_form' do
-      if RUBY_VERSION =~ /2.7/
-        expect(PdfFill::Filler).to receive(:fill_form).with(claim, claim.guid, {}).once.and_return(:expected_file_paths)
-      else
-        expect(PdfFill::Filler).to receive(:fill_form).with(claim, claim.guid).once.and_return(:expected_file_paths)
-      end
+      expect(PdfFill::Filler).to receive(:fill_form).with(claim, claim.guid).once.and_return(:expected_file_paths)
       expect(claim.to_pdf).to eq(:expected_file_paths)
     end
 
-    it 'passes arguments to PdfFill::Filler#fill_form' do
-      if RUBY_VERSION =~ /2.7/
+    context 'passes arguments to PdfFill::Filler#fill_form' do
+      it 'only the file name' do
         expect(PdfFill::Filler).to receive(
           :fill_form
         ).with(
           claim,
-          'my_other_filename',
-          {}
+          'my_other_filename'
         ).once.and_return(:expected_file_paths)
-      else
+
+        # Calling with only filename
+        claim.to_pdf('my_other_filename')
+      end
```

Comment:
🎨 the `it` blocks here are somewhat awkwardly phrased? maybe something like
`it 'converts to pdf with the file name alone' do`
?

regardless, breaking it down into these discrete `it` blocks neatly makes the comments superfluous, which is a nice readability win!

Reply:
Yeah I wasn't sure how to best word it. I can update it.